public/javascripts/tests.js

```diff
@@ -0,0 +1,96 @@
+function renderTestsList(jobs, is_operator, restart_url) {
+
+    var table = $('#results').DataTable( {
+	"dom": 'l<"#toolbar">frtip',
+	"lengthMenu": [[10, 25, 50], [10, 25, 50]],
+	"ajax": {
+	    "url": "/tests/list_ajax",
+	    "type": "POST", // we use POST as the URLs can get long
+	    "data": function(d) {
+		var ret = {
+		    "relevant": $('#relevantfilter').prop('checked')
+		};
+		if (jobs != null) {
+		    ret['jobs'] = jobs;
+		    ret['initial'] = 1;
+		}
+		// reset for reload
+		jobs = null;
+		return ret;
+	    }
+	},
+	// no initial resorting
+	"order": [],
+	"columns": [
+	    { "data": "name" },
+	    { "data": "test" },
+	    { "data": "deps" },
+	    { "data": "testtime" },
+	    { "data": "result_stats" },
+	],
+	"columnDefs": [
+	    { targets: 0,
+	      className: "name",
+	      "render": function ( data, type, row ) {
+		  var name = 'Build' + row['build'];
+		  name += " of ";
+		  return name + row['distri'] + "-" + row['flavor'] + "." + row['arch'];
+	      },
+	    },
+	    { targets: 1,
+	      className: "test",
+	      "render": function ( data, type, row ) {
+		  if (type === 'display') {
+		      html = '<a class="overview_' + row['result'] + '" href="tests/' + row['id'] + '">' + data + '</a>';
+		      if (row['clone']) {
+                          html += ' <a href="/tests/' + row['clone'] + '">(restarted)</a>';
+                      } else {
+			  var url = restart_url.replace('REPLACEIT', row['id']);
+                          html += ' <a data-remote="true" href="' + url + '">' +
```

Comment:
you're right here too - and I noticed another problem while writing the test: the icon shouldn't be displayed unless logged in